microservicecloud/microservicecloud-provider-dept-hystrix-8002/src/test/java/tests/RunIT.java

```diff
@@ -0,0 +1,59 @@
+package tests;
+
+import com.gof.springcloud.TravePlanProvider8002_App;
+import org.junit.Assert;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.core.env.Environment;
+import org.springframework.http.*;
+
+@SpringBootTest(
+        classes = TravePlanProvider8002_App.class,
+        webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT
+)
+public class RunIT {
```

Comment:
```suggestion
public class RunIT {
```

Able to make update to api test? Test elasticsearch apis